src/tests/docs/t_conduit_docs_tutorial_python_basics.py

```diff
@@ -92,4 +92,74 @@ def test_007_basics_bw_style_from_native(self):
         print(n.schema())
         END_EXAMPLE("py_basics_bw_style_from_native")
 
+    def test_008_basics_numpy_or_node(self):
+        BEGIN_EXAMPLE("py_basics_numpy_or_node")
+        # setup a node with a leaf array
+        n = conduit.Node()
+        data = numpy.zeros((5,),dtype=numpy.float64)
+        n["my/path/to/data"] = data
+        
+        # this will be a ndarray
+        my_data = n["my/path/to/data"]
+        print("== this will be a ndarray == ")
```

Comment:
typo: should be "this will be an ndarray"   (fix also lines 102, 125; line 123 has "an ndarray" already)